Sources/TuistGraph/Models/Target.swift

```diff
@@ -203,6 +203,14 @@ public struct Target: Equatable, Hashable, Comparable, Codable {
         destinations.first?.platform ?? .iOS
     }
 
+    /// Platform for use with commands which assume single platform targets
+    public var exclusivePlatform: Platform? {
```

Comment:
I was a bit confused when reading "exclusive". What about calling it `platformForAutomation` or `platformIfOnlyOne`? It's a bit more verbose, but more explicit when one reads it from the context of the caller.

Reply:
pushed a change moving this logic to an extension in `TuistKit` focused just on services